docs/querying/lookups-cached-global.md

```diff
@@ -210,43 +214,48 @@ The remapping values for each globally cached lookup can be specified by a JSON
 
 |Property|Description|Required|Default|
 |--------|-----------|--------|-------|
-|`pollPeriod`|Period between polling for updates|No|0 (only once)|
-|`uri`|URI for the lookup file. Can be a file, HDFS, S3 or GCS path|Either `uri` or `uriPrefix` must be set|None|
-|`uriPrefix`|A URI prefix that specifies a directory or other searchable resource where lookup files are located |Either `uri` or `uriPrefix` must be set|None|
-|`fileRegex`|Optional regex for matching the file name under `uriPrefix`. Only used if `uriPrefix` is used|No|`".*"`|
-|`namespaceParseSpec`|How to interpret the data at the URI|Yes||
-|`maxHeapPercentage`|The maximum percentage of heap size that the lookup should consume. If the lookup grows beyond this size, warning messages will be logged in the respective service logs.|No|10% of JVM heap size|
+|`pollPeriod`|Time period between polling for updates, in ms.|No|0 (only once)|
+|`uri`|URI for the lookup file. Can be a file, HDFS, S3 or GCS path.|You must define either `uri` or `uriPrefix`.*|None|
+|`uriPrefix`|A URI prefix that specifies a directory or other searchable resource where lookup files are located.|You must define either `uri` or `uriPrefix`.*|None|
+|`fileRegex`|Optional regex for matching the file name under `uriPrefix`. Only used if `uriPrefix` is defined.|No|`".*"`|
+|`namespaceParseSpec`|How to interpret the data at the URI.|Yes||
+|`maxHeapPercentage`|The maximum percentage of heap size for the lookup to consume. If the lookup grows beyond this size, Druid logs warning messages in the respective service logs.|No|10% of JVM heap size|
 
-One of either `uri` or `uriPrefix` must be specified, as either a local file system (file://), HDFS (hdfs://), S3 (s3://) or GCS (gs://) location. HTTP location is not currently supported.
+*Define `uri` or `uriPrefix` as either a local file system (`file://`), HDFS (`hdfs://`), S3 (`s3://`) or GCS (`gs://`) location. Druid doesn't support HTTP location.
 
-The `pollPeriod` value specifies the period in ISO 8601 format between checks for replacement data for the lookup. If the source of the lookup is capable of providing a timestamp, the lookup will only be updated if it has changed since the prior tick of `pollPeriod`. A value of 0, an absent parameter, or `null` all mean populate once and do not attempt to look for new data later. Whenever an poll occurs, the updating system will look for a file with the most recent timestamp and assume that one with the most recent data set, replacing the local cache of the lookup data.
+The `pollPeriod` value specifies the period in ISO 8601 format between checks for replacement data for the lookup. If the source of the lookup is capable of providing a timestamp, the lookup is only updated if it has changed since the prior tick of `pollPeriod`. A value of 0, an absent parameter, or `null` all direct Druid to populate the lookup once and not attempt to look for new data later. 
```

Comment:
```suggestion
The `pollPeriod` value specifies the period in ISO 8601 format between checks for replacement data for the lookup. If the source of the lookup provides a timestamp, Druid only updates the lookup when the lookup has changed since the prior `pollPeriod` timestamp. A value of 0, an absent parameter, or `null` all direct Druid to populate the lookup once without attempting to check again for new data. 
```